settings.gradle

```diff
@@ -98,7 +98,7 @@ include(':jballerina-integration-test')
 // and 'mustRunAfter' of vs-code-pluggin
 
 include(':plugin-vscode')
-include(':ballerina-intellij-idea-plugin')
+// include(':ballerina-intellij-idea-plugin')
```

Comment:
If this detachment is permanent, better to remove these commented lines as well.

Reply:
I left it intentionally in case we need to enable the plugin build. But now I also think its better to be removed. Done